### ISSUE TITLE
Problem with outline-width getting dropped sometimes

### ISSUE THREAD

**Issue by hostep** [jakubpawlowicz/clean-css#541]
This is ok:
```
➜ echo 'button:focus { outline: 5px auto -webkit-focus-ring-color; outline-offset: -2px; }' | ./cleancss
button:focus{outline:-webkit-focus-ring-color auto 5px;outline-offset:-2px}
```

This is not ok (the 5px is getting dropped):
```
➜ echo 'button:focus { outline: 5px auto -webkit-focus-ring-color; } button:focus { outline-offset: -2px; }' | ./cleancss
button:focus{outline:-webkit-focus-ring-color auto;outline-offset:-2px}
```

Using clean-css 3.2.2, it did work before in 3.1.9.

**Comment by jakubpawlowicz** [comment]
That's a bug indeed. Stay tuned for a fix!

**Comment by jakubpawlowicz** [comment]
Fixed in 3.2.3!

**Comment by hostep** [comment]
Thanks once again! :)
